docs/README.md

```diff
@@ -1,27 +1,8 @@
 # NGINX Gateway Fabric Documentation
 
-This directory contains all of the documentation relating to NGINX Gateway Fabric.
+This directory contains the developer documentation and the Enhancement proposals relating to NGINX Gateway Fabric.
```

Comment:
Since many users are probably accustomed to the public docs being in this folder, it may be worth adding a note that they have been moved to docs.nginx.com, as well as ensuring that our top level README links there as well.

Reply:
- https://github.com/nginxinc/nginx-gateway-fabric/blob/main/README.md#getting-started
- Table for edge entry and main branch (not the release entry just yet, since that branch still uses the old docs for now) https://github.com/nginxinc/nginx-gateway-fabric/blob/main/README.md#nginx-gateway-fabric-releases

Probably worth having a nice note near the top of that README as well that just links straight to the main docs page.